support-services/src/main/scala/com/gu/support/catalog/CatalogService.scala

```diff
@@ -14,6 +14,34 @@ object CatalogService {
 }
 
 class CatalogService(val environment: TouchPointEnvironment, jsonProvider: CatalogJsonProvider) extends LazyLogging {
+
+  private def getRatePlanId(billingPeriod: BillingPeriod, fulfilmentOptions: FulfilmentOptions)
+    = getProductRatePlan(environment, billingPeriod,fulfilmentOptions, NoProductOptions).map(_.id).getOrElse("")
+
+  private def fetchQuarterlyPrice(quarterlyId: ProductRatePlanId, sixWeeklyPriceList: Pricelist, catalogPrices: List[Pricelist]) = {
```

Comment:
Line too long, we should do each argument on a line. Also same feedback as above, we may only need `private[this]`.